### DISCUSSION TITLE
tox/docs/config.rst - temp_dir possible change

### DISCUSSION THREAD

**Discussion by cesarcoatl** [tox-dev/tox#2930] [General]
Hello,

In one of my projects where I ran `tox` I got an error from the IDE I was using saying that `.tmp` was not valid, and it turned out that it was referring to `.tox/.tmp`.

When I wanted to change the default name for `temp_dir` via `tox.ini` I noticed a potential mistake in the docs.

I've forked this repo and this is the `diff` for the change I might propose in a PR if it turns out I'm right.

```sh
$ git diff
diff --git a/docs/config.rst b/docs/config.rst
index adc7863..3804acd 100644
--- a/docs/config.rst
+++ b/docs/config.rst
@@ -174,7 +174,7 @@ Core
 
 .. conf::
    :keys: temp_dir
-   :default: {tox_root}/.tmp
+   :default: {work_dir}/.tmp
 
    Directory where to put tox temporary files. For example: we create a hard link (if possible, otherwise new copy) in
    this directory for the project package. This ensures tox works correctly when having parallel runs (as each session
```

>`tox_root` `toxinidir`
>
>The root directory for the tox project (where the configuration file is found).

While...

>`work_dir` `toxworkdir` with default value of `{tox_root}/.tox`

So as my `diff` shows, I believe the docs should be modified.

**Comment by jugmac00** [comment]
Sounds right! Thanks for spotting. I am happy to review / merge your merge proposal.

**Reply by cesarcoatl** [reply]
Thanks, @jugmac00.

Here's the PR: https://github.com/tox-dev/tox/pull/2932
